maven-scm-api/src/main/java/org/apache/maven/scm/command/untag/AbstractUntagCommand.java

```diff
@@ -31,18 +31,27 @@
 public abstract class AbstractUntagCommand
     extends AbstractCommand
 {
+    /**
+     * SCM-917 - scm:untag for subversion
+     * @param repository scm repo
+     * @param fileSet set of files (unused)
+     * @param tagName tag name to remove
+     * @param message to use for commit comment
+     * @return result of untag command
+     * @throws ScmException  in case of error
+     */
     protected abstract ScmResult executeUntagCommand( ScmProviderRepository repository,
-        ScmFileSet fileSet, String tagName )
-        throws ScmException;
+        ScmFileSet fileSet, String tagName, String message ) throws ScmException;
```

Comment:
Please reformat.

Reply:
Why aren't you using the same approach as Tag does with `ScmTagParameters scmTagParameters` instead of the pure message? It would look consistent in my opinion.

Reply:
OK.  Used untag parameters now.